src/Sweep.cpp

```diff
@@ -282,7 +282,7 @@ void InfectSweep(double t, int run) //added run number as argument in order to r
 	int f, f2, cq /*cell queue*/, bm, ci /*person index*/;
 	double seasonality, sbeta, hbeta;
 	//// various quantities of force of infection, including "infectiousness" and "susceptibility" components
-	double s; // household FOI on fellow household member, then place susceptibility, then random number for spatial infections allocation* / ;
+	double s; // household Force Of Infection (FOI) on fellow household member, then place susceptibility, then random number for spatial infections allocation* / ;
```

Comment:
I think the variable should be split into three variables, instead of re-used. 
Then, each variable should be named appropriately.

Reply:
I don't disagree. My initial approach was not to make any code changes but just add comments to aid understanding of the code 'as is' with the hope that this would aid refactoring.

Reply:
I figured something like that. In that case, I approve. :)